pkg/vm/descriptors.go

```diff
@@ -4,19 +4,29 @@ import (
 	"math/big"
 
 	"github.com/ava-labs/avalanche-cli/cmd/prompts"
+	"github.com/ava-labs/avalanche-cli/pkg/app"
 	"github.com/ava-labs/avalanche-cli/ux"
 )
 
-func getChainId() (*big.Int, error) {
+func getChainId(app *app.Avalanche) (*big.Int, error) {
 	// TODO check against known chain ids and provide warning
 	ux.Logger.PrintToUser("Enter your subnet's ChainId. It can be any positive integer.")
 
-	chainId, err := prompts.CapturePositiveBigInt("ChainId")
+	chainID, err := prompts.CapturePositiveBigInt("ChainId")
```

Comment:
I prefer `ChainId` over `ChainID`. We should adopt a universal standard there.

Reply:
I believe it is `chainId` in the actual Eth-RPC spec: https://eips.ethereum.org/EIPS/eip-695

Reply:
I think it's a golang standard to use uppercase letters for such cases:
https://github.com/golang/go/wiki/CodeReviewComments#initialisms mentions ID explicitly.

I think we have applied this scheme in all other our golang repos.

Reply:
I don't love it, but I can live with it, I'll open a ticket to convert the other references.